project/Dependencies.scala

```diff
@@ -23,7 +23,9 @@ object Dependencies {
      // For Twitter's util functions
     "Twitter Maven Repo"             at "https://maven.twttr.com/",
     // For legacy Snowplow libs
-    ("Snowplow Analytics Maven repo" at "http://maven.snplow.com/releases/").withAllowInsecureProtocol(true)
+    ("Snowplow Analytics Maven repo" at "http://maven.snplow.com/releases/").withAllowInsecureProtocol(true),
```

Comment:
I know it's not connected to the current PR, but do we still need the Snowplow Maven repo?  Which libs are we pulling?

Reply:
probably not, let me check if we are fine without it

Reply:
It must be a Thrift-based `CollectorPayload`

Reply:
from coursier output

```
Downloading http://maven.snplow.com/releases/com/snowplowanalytics/snowplow-thrift-raw-event/0.1.0/
Downloading http://maven.snplow.com/releases/com/snowplowanalytics/schema-sniffer-1/0.0.0/
Downloading http://maven.snplow.com/releases/com/snowplowanalytics/collector-payload-1/0.0.0/
```